### ISSUE TITLE
Spanish Tracker

### ISSUE THREAD

**Issue by ghost** [BurningMop/qBittorrent-Search-Plugins#15]
I was wondering if tomadivx is still pending or if you haven't been able to find any trackers that have Spanish content.

**Comment by BurningMop** [comment]
Yes, is still pending. I haven't had much free time to work on it. I did however, push some code to qBitTorrent repo with some functionality I need (I could probably help other developers).

 If you know any other spanish tracker, feel free to add it here, and will try to work on it

**Comment by ghost** [comment]
I give you a list of the best that exist: 

https://wolfmax4k.com

https://www.pelitorrent.com 

Then there are clones of DonTorrent that would be good to make work because the DonTorrent plugin fails in searches: 

https://esmeraldatorrent.com

https://naranjatorrent.com 

https://divxtotal.wtf 

Others that are based on their database in case the searches I have done fail too:

https://calidadtorrent.com

https://pediatorrent.com

**Comment by BurningMop** [comment]
I'll do my best, I'm in the middle of moving to a new house, some time is kind of tight

**Comment by BurningMop** [comment]
Hola @ghost, le eché un ojo a los buscadores, y el único que veo complicado es wolfmax4k, el resto parecen posibles. Ya lo agregué en el readme del repo

**Comment by BurningMop** [comment]
Hola @ghost, el plugin de dontorrent (y sus clones) y el de pelitorrent (ahora se llaman torrenflix) ya estan disponibles.

**Comment by BurningMop** [comment]
Hola de nuevo, excepto Wolfmax, el resto ya estan disponibles para su uso.

**Comment by WoRMx00** [comment]
hola @BurningMop , estuve probando varios plugins para la versión 5.0 de qbt y en todos me dice que no están soportados. Uso la ultima versión 5.0.2
